buildSrc/build.gradle.kts

```diff
@@ -18,46 +18,47 @@
 
 // Plugins for configuring _this build_ of the module
 plugins {
-  id 'java-gradle-plugin'
-  id 'groovy'
-  id "com.diffplug.spotless" version "5.6.1"
+  `java-gradle-plugin`
+  groovy
+  id("com.diffplug.spotless") version "5.6.1"
+  `kotlin-dsl`
```

Comment:
Just curious, why do we need the `kotlin-dsl` plugin here but not in the other files?

Reply:
Removed.

It is needed for Kotlin support in `buildSrc` project.
It will be needed if some of the existing Groovy Beam plugins going to be migrated to Kotlin. (Which I tried to do and left it in the PR by accident, thank you)